package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A simple Telegram <=> IRC bot",
   "main": "teleirc.js",
   "scripts": {
+    "start": "node teleirc.js",
```

Comment:
On a side note, it could be an interesting side-project for us to package an RPM for TeleIRC. A future sprint idea.